uninstall.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * performance-lab uninstall method
```

Comment:
Let's add the `@package` here like in the other files.
```suggestion
 * Plugin ninstaller logic.
 *
 * @package performance-lab
 * @since n.e.x.t
```

Reply:
Resolved, thank you